filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -937,8 +937,36 @@ void OpenGLDriver::framebufferTexture(backend::TargetBufferInfo const& binfo,
         attachmentTypeNotSupportedByMSRTT = true;
     }
 
-    if (rt->gl.samples <= 1 ||
-        (rt->gl.samples > 1 && t->samples > 1 && gl.features.multisample_texture)) {
+    // The following 4 techniques are different mechanisms to provide storage to the FBO.
+    // We'll use the first technique that is sufficient.
+    // nonMSAA:         MSAA was not requested, or we're rendering to an MS texture which
+    //                  doesn't require any special setup here
+    // useMSRTT:        use the EXT_multisampled_render_to_texture extension
```

Comment:
Similarly, this is more "useMsrttMsaaRsolve". i.e. MSAA resolve will be handled by MSRTT.